### PR TITLE
Use Origin instead of Referer

### PR DESCRIPTION
**Pull request by npm1**
Fixes https://github.com/fedidcg/FedCM/issues/379
cc @bvandersloot-mozilla as not sure what will be the right way to ask the Fetch spec to set the Origin header vs not set it, in the new fetch mode introduced in FedCM.

***
<a href="https://pr-preview.s3.amazonaws.com/npm1/FedCM/pull/423.html" title="Last updated on Feb 9, 2023, 3:52 PM UTC (2b3a614)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/423/97004a2...npm1:2b3a614.html" title="Last updated on Feb 9, 2023, 3:52 PM UTC (2b3a614)">Diff</a>

### CODE REVIEW

**Review by samuelgoto — APPROVED** [review]
LGTM

@bvandersloot-mozilla LGTY?

File: spec/index.bs [hunk 1]
```diff
@@ -319,7 +319,7 @@ The well-known file is fetched:
 
 (a) **without** cookies,
 (b) **with** the <a http-header>Sec-Fetch-Dest</a> header set to `webidentity`, and
-(c) **without** a [[RFC9110#field.referer|Referer]] header
```
Comment (bvandersloot-mozilla) [inline spec/index.bs:322]: Genuine question: do we want to include that the Referer also will not be sent here (and in similar lists)?

File: spec/index.bs [hunk 2]
```diff
@@ -1191,12 +1188,10 @@ To <dfn>fetch an identity assertion</dfn> given an [=AccountState=] |accountStat
         :  [=request/destination=]
         :: "webidentity"
         :  [=request/origin=]
-        :: a unique [=opaque origin=]
+        :: [=RP=]'s [=/origin=] (TODO)
```
Comment (bvandersloot-mozilla) [inline spec/index.bs:1191]: This works for now. Suggestion: replace the `(TODO)` with an inline `ISSUE: ` to switch to `cors-mode: user-agent-no-cors` when available?

**Review by bvandersloot-mozilla — APPROVED** [review]
Looks pretty good. One suggestion and one question that can be resolved as you see fit.

File: spec/index.bs (hunk 2 above)
Comment (npm1) [reply spec/index.bs:1194]: There is already an issue noted for this. I updated the string to `user-agent-no-cors`

**Pull request merged**